### DISCUSSION TITLE
Async Api doesn't resolve

### DISCUSSION THREAD

**Discussion by momolafrooo** [infinitered/ignite#1763] [General]
Hello Guys, i'm new on ignite. I'm trying to make an api call with laravel sanctum as authentication. But i have an issue with api sauce. The async call doesn't resolve when i try to add authorization to the header.
Can someone help me ?

**Comment by jamonholmgren** [comment]
@momolafrooo Did you get this figured out? It's pretty difficult to say what might be wrong without more details.

**Comment by danielminhnguyen** [comment]
It happened to me also. Turn out it was because of the hot reload. It run ok at first, but once reload (press r on metro) the promise won't resolve. Deleted the app on simulator and reinstall helps.

**Comment by momolafrooo** [comment]
Hello guys I finally solved the problem by removing de ToggleStoryBook component. 
The problem comes from this component.

**Comment by danielminhnguyen** [comment]
I tried to disable the ToggleStoryBook and the problem seems gone too. Thanks @momolafrooo

**Reply by jamonholmgren** [reply]
We are aware of this issue, sorry about that. Working on it over on issue #1863
